src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
+Animal.removeAnimal = function(objToRemove) {
+  delete this.alive[Animal.alive.indexOf(objToRemove)];
```

Comment:
better to store something in the constants, as it now does not appear to be readable

Reply:
also, do you believe `objToRemove` is a good name for a variable?